docs/deprecated/functionalities.rst

```diff
@@ -1,5 +1,5 @@
 "Eggsecutable" Scripts
-----------------------
+=================
```

Comment:
I recall that rst is strict about heading length matches, so will likely warn here (and fail in nitpicky mode). Let's keep the length consistent.